ingestion_tools/scripts/data_validation/fixtures/path.py

```diff
@@ -70,28 +70,61 @@ def run_meta_file(run_dir: str, filesystem: FileSystemApi) -> str:
 
 
 @pytest.fixture(scope="session")
-def frames_dir(run_dir: str, tiltseries_metadata: Dict[str, Any], filesystem: FileSystemApi) -> str:
+def frames_dir(run_dir: str, filesystem: FileSystemApi) -> str:
     """[Dataset]/[ExperimentRun]/Frames"""
     dst = f"{run_dir}/Frames"
-    if filesystem.s3fs.exists(dst):
+    if filesystem.exists(dst):
         return dst
     else:
-        if tiltseries_metadata["frames_count"] > 0:
-            pytest.fail(f"Frames directory not present: {dst}")
-        else:
-            pytest.skip(f"Frames directory not present: {dst}")
+        pytest.skip(f"Frames directory not present: {dst}")
+
+
+@pytest.fixture(scope="session")
+def frames_files(frames_dir: str, filesystem: FileSystemApi) -> List[str]:
+    """[Dataset]/[ExperimentRun]/Frames/[frame_name].mrc"""
+    files = filesystem.glob(f"{frames_dir}/*")
+    return ["s3://" + file for file in files if "_gain" not in file]  # Exclude gain files
 
 
 @pytest.fixture(scope="session")
 def frame_acquisition_order_file(frames_dir: str, filesystem: FileSystemApi) -> str:
     """[Dataset]/[ExperimentRun]/Frames/frame_acquisition_order.json"""
     dst = f"{frames_dir}/frame_acquisition_order.json"
```

Comment:
We already retrieve the tiltseries mdoc file as of now, and this fixture is just here for the future (I believe we said in a discussion that it would be created sometime in the future). But should we remove it for now?